frontend/migrations/versions/f524c4a47236_add_user_name_to_completed_challenge_table.py

```diff
@@ -0,0 +1,28 @@
+"""add user_name to completed_challenge table
+
+Revision ID: f524c4a47236
+Revises: 74f03c8be8dd
+Create Date: 2022-01-17 13:08:53.794787
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'f524c4a47236'
+down_revision = '74f03c8be8dd'
```

Comment:
Why we are using the old revision numbers? @adityaevontech when you copy and paste the revions please change the revison numbers

Reply:
These revision numbers are generated by Alembic '74f03c8be8dd' is the old migration that was last performed.

Reply:
Kindly make/push docker_ui and docker_frontend tagged as latest

Reply:
why latest Aditya ?